packages/itwin/grouping-mapping-widget/src/widget/components/customUI/DefaultGroupingUI.tsx

```diff
@@ -27,7 +27,7 @@ export const defaultGroupingUI: GroupingMappingCustomUI[] = [
   },
   {
     name: "Manual",
-    displayLabel: "Manual",
+    displayLabel: "Manual Query",
```

Comment:
Why was this renamed?

Reply:
I saw in both the backlog item image for this bug (there was a "..." following "Manual") and the readme file under the grouping-mapping-widget folder where it was specified as "Manual Query". I think it is clearer too now that we have space, but I should have asked first.

Reply:
No worries. I think I renamed it to Manuel Query when I did a whole refactor of this widget then it was caught my QA asking the same question, I believe they took that screenshot when I made that mistake.

Reply:
Yes